lib/MadNLPGPU/src/cudss.jl

```diff
@@ -20,6 +20,7 @@ function CUDSSSolver(
     csc::CUSPARSE.CuSparseMatrixCSC{T};
     opt=CudssSolverOptions(),
     logger=MadNLP.MadNLPLogger(),
+    ordering::Symbol=:default
```

Comment:
This should be moved to `CXudssSolverOptions` to be exposed to the users

Reply:
And maybe we can use https://github.com/MadNLP/MadNLP.jl/blob/master/lib/MadNLPGPU/src/cusolverrf.jl#L184-L187
rather than symbols

Reply:
The issue is that `CUCHOLESKYORDERING` has only two values. We can't add other ordering like `symamd` or `colamd` nor use the default ordering that is computed by cuDSS.